test/e2e/common/security_context.go

```diff
@@ -130,21 +129,24 @@ var _ = framework.KubeDescribe("Security Context", func() {
 		}
 
 		/*
-		  Release : v1.12
-		  Testname: Security Context: readOnlyRootFilesystem=true.
-		  Description: when a container has configured readOnlyRootFilesystem to true, write operations are not allowed.
-		  This test is marked LinuxOnly since Windows does not support creating containers with read-only access.
+			Release : v1.15
+			Testname: Security Context, readOnlyRootFilesystem=true.
+			Description: Container is configured to run with readOnlyRootFilesystem to true which will force containers to run with a read only root file system.
+			Write Operation MUST NOT be allowed and Pod MUST be in Failed state.
+			At this moment we are not considering this test for Conformance due to use of SecurityContext.
+			[LinuxOnly]: This test is marked as LinuxOnly since Windows does not support creating containers with read-only access.
 		*/
 		It("should run the container with readonly rootfs when readOnlyRootFilesystem=true [LinuxOnly] [NodeConformance]", func() {
 			createAndWaitUserPod(true)
 		})
 
 		/*
-		  Release : v1.12
-		  Testname: Security Context: readOnlyRootFilesystem=false.
-		  Description: when a container has configured readOnlyRootFilesystem to false, write operations are allowed.
+			Release : v1.15
+			Testname: Security Context, readOnlyRootFilesystem=false.
+			Description: Container is configured to run with readOnlyRootFilesystem to false.
+			Write Operation MUST be allowed and Pod MUST be in Succeeded state.
 		*/
-		It("should run the container with writable rootfs when readOnlyRootFilesystem=false [NodeConformance]", func() {
+		framework.ConformanceIt("should run the container with writable rootfs when readOnlyRootFilesystem=false [NodeConformance]", func() {
```

Comment:
I like how you indicated which field exactly is set to false here (`readOnlyRootFilesystem`), can we do the same for all the other test cases we're touching in this file?

Reply:
I have added this format in Conformance doc but not changed e2e name as this PR is intended to promote existing e2e to Conformance and would require another PR to change such e2e updates. 

Sometimes it gives good readibity by either `readOnlyRootFilesystem` or ` when Filesystem is readOnly` based on Context(). IMO, both are good approach and such change would be good if it improves readability whereever required or if we are planning to enforce using `$propery=value` in e2e name.

FYI, Many e2e names are not precisely defined based on their behavior so we are updating them i.e. https://github.com/kubernetes/kubernetes/pull/75544, #75632.